src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -87,7 +84,8 @@ public ProjectsListViewController(final Model model, final Controller controller
 
             final String lowerCaseFilter = newValue.toLowerCase();
 
-            if (project.getName().toLowerCase().contains(lowerCaseFilter)) {
+            if (project.getName().toLowerCase().contains(lowerCaseFilter)
```

Comment:
did you write a test for the new filter logic?

Reply:
Done.